WebVTT: Test the extremes for parsing line: setting

zcorpan opened the pull request:
Follows https://github.com/w3c/webvtt/pull/339.

The test already expected double precision float representation
with the "10000000000000000000000000000000000" test.

The "Greater than Number.MAX_VALUE" test tests the case where the
rules for parsing floating-point number values algorithm returns an
error.

---

cc @BenjaminSchaaf @fsoder

zcorpan committed:
WebVTT: Test the extremes for parsing line: setting

Follows https://github.com/w3c/webvtt/pull/339.

The test already expected double precision float representation
with the "10000000000000000000000000000000000" test.

The "Greater than Number.MAX_VALUE" test tests the case where the
rules for parsing floating-point number values algorithm returns an
error.

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/208541022)

# Firefox (nightly channel) #
Testing web-platform-tests at revision a905e8c9a9fc2d4c039cf2634fc4527bec0afa35
Using browser at version BuildID 20170306110339; SourceStamp 966464a68a2cb3ca1125808e34abb5c1d34e3797
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5062/webvtt/parsing/file-parsing/tests/settings-line.html">/webvtt/parsing/file-parsing/tests/settings-line.html</a></summary>


|     Subtest      | Results |                                           Messages                                          |
|------------------|---------|---------------------------------------------------------------------------------------------|
|                  | OK      |                                                                                             |
| `settings, line` | FAIL    | `assert_unreached: got unexpected error event: {"isTrusted":true} Reached unreachable code` |

</details>
</details>

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/208541023)

# Chrome (unstable channel) #
Testing web-platform-tests at revision a905e8c9a9fc2d4c039cf2634fc4527bec0afa35
Using browser at version 58.0.3026.3 dev
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5062/webvtt/parsing/file-parsing/tests/settings-line.html">/webvtt/parsing/file-parsing/tests/settings-line.html</a></summary>


|     Subtest      | Results |                                  Messages                                 |
|------------------|---------|---------------------------------------------------------------------------|
|                  | OK      |                                                                           |
| `settings, line` | FAIL    | `assert_equals: Failed with cue 7 expected 4294967296 but got 2147483648` |

</details>
</details>

zcorpan committed:
Remove confusing JSON.stringify(errorEvent)

The JSON serialization of the error event is `{"isTrusted":true}`
which is more confusing than helpful.

zcorpan commented:
https://bugzilla.mozilla.org/show_bug.cgi?id=1345076
https://bugs.chromium.org/p/chromium/issues/detail?id=699063
